src/styles/header.scss

```diff
@@ -0,0 +1,68 @@
+@import 'fonts';
+@import 'colors';
+
+.containerHeader {
+  display: flex;
+  width: auto;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  font-family: $roboto-medium;
+  font-weight: $roboto-weight-medium;
```

Comment:
The font-family and font-weight are set with variables, which is good for consistency. However, ensure that these variables are declared in the 'fonts.scss' and 'measures.scss' files respectively, as per the project structure.